yarn.lock

```diff
@@ -5359,11 +5359,6 @@ clone-response@^1.0.2:
   dependencies:
     mimic-response "^1.0.0"
 
-clone@2.x:
```

Comment:
Do we need to update yarn.lock in this PR?

Reply:
Possibly not! Thanks Rupert